docs/changelog.rst

```diff
@@ -20,6 +20,7 @@ Minor Changes:
     - Added polish to missing value visualizers code, tests, and documentation.
     - Improved RankD tests for better coverage.
     - Added quick method test for ``DispersionPlot`` visualizer.
+    - Extended FeatureImportances documentation and tests for stacked importances and added a warning when stack should be true.
```

Comment:
Trying to update the changelog with new commits to make releases easier.

Reply:
really appreciate you for doing this..I have to make sure I do it also